app/uk/gov/hmrc/helptosavefrontend/connectors/NSIConnector.scala

```diff
@@ -70,44 +68,55 @@ class NSIConnectorImpl @Inject() (conf: Configuration, auditor: HTSAuditor, metr
       ()
     )
 
-    val timeContext: Timer.Context = timer.time()
+    val timeContext: Timer.Context = metrics.nsiAccountCreationTimer.time()
 
     httpProxy.post(nsiUrl, userInfo, Map(nsiAuthHeaderKey → nsiBasicAuth))
       .map[SubmissionResult] { response ⇒
         val time = timeContext.stop()
 
         response.status match {
           case Status.CREATED ⇒
-            auditor.sendEvent(new ApplicationSubmittedEvent(appName, userInfo))
+            auditor.sendEvent(ApplicationSubmittedEvent(appName, userInfo))
```

Comment:
can we add counters for success cases as well ?

Reply:
we can derive the success cases by doing the count of times timed minus the error count so we don't need an explicit success counter